README.md

```diff
@@ -1,24 +1,73 @@
-# README
+## Hotel Reservstion 🏩
+This application is a web application for final capstone project that you can find your favorite Hotel around the world and reserve it for a specific date, find information about a hotel at details page and cancel a reservation.
```

Comment:
- [ ] You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍